src/scheduler/GUI.java

```diff
@@ -751,12 +758,75 @@ private void buildDateGUI() {
         btnCreateEvent.setFont(btnUCreateEvent.getFont());
         btnCreateEvent.setForeground(btnUCreateEvent.getForeground());
 
-        upperPanel.add(btnBack);
-        upperPanel.add(btnCreateEvent);
-        lowerPanel.add(new javax.swing.JLabel("Time"));
-        lowerPanel.add(new javax.swing.JLabel("Events"));
-        dateGUI.add(upperPanel);
-        dateGUI.add(lowerPanel);
+        //layout
+        c.insets = new java.awt.Insets(2, 2, 4, 4);
+        c.fill = GridBagConstraints.HORIZONTAL;
+        c.gridx = 0;
+        c.gridy = 0;
+        c.anchor = GridBagConstraints.NORTHWEST;
+        upperPanel.add(btnBack, c);
+        
+        c.gridx = 1;
+        c.gridy = 0;
+        c.anchor = GridBagConstraints.NORTH;
+        upperPanel.add(btnCreateEvent, c);
+        
+        c.gridx = 0;
+        c.gridy = 1;
+        c.anchor = GridBagConstraints.NORTHEAST;
+        c.fill = GridBagConstraints.NONE;
+        lowerPanel.add(new javax.swing.JLabel("Time"), c);
+        
+        c.gridx = 1;
+        c.gridy = 1;
+        c.gridwidth = 3;
+        c.anchor = GridBagConstraints.CENTER;
+        c.weightx = 1;
+        lowerPanel.add(new javax.swing.JLabel("Events"), c);
+        
+        //fill in times and event descriptions
```

Comment:
Is this pulling from events ? From what I see, it just has time and has an increment of 30... what is the purpose ? 
or intended functionality ?